src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -148,20 +151,68 @@ bool VisitAggregateAttribute(ITypeSymbol _attributeSymbol,int depth)
 			}
 		}
 
-		public bool IsBoundAttribute(ITypeSymbol attributeSymbol)
+		public BoundFlag IsBoundAttribute(AttributeData attribute)
 		{
 			foreach (var baseType in BoundBaseTypes)
 			{
-				if (AttributeDerivedFromClass(attributeSymbol, baseType))
+				if (AttributeDerivedFromClass(attribute.AttributeClass, baseType))
+					return BoundFlag.DbBound;
+			}
+
+			if (attribute.AttributeClass.GetMembers().Select(a => a.Name).Contains(IsDBField))
+			{
+				foreach (var argument in attribute.NamedArguments)
+				{
+					if (argument.Key.Equals(IsDBField))
+					{
+						if (argument.Value.Value.Equals(true))
+							return BoundFlag.DbBound;
+						else
+							return BoundFlag.Unbound;
+					}
+				}
+				return BoundFlag.Unknown;
+			}
+			return BoundFlag.Unbound;
+		}
+
+		///TODO: refactoring arguments -> remove semanticModel to constructor? Is it nessesary?
+		///TODO: Add DataFlow analyze to corner cases with defaul IsDBBound assigment.
+		public bool? IsBoundField(PropertyDeclarationSyntax property, SemanticModel semanticModel)
+		{
+			var typeSymbol = semanticModel.GetDeclaredSymbol(property);
+			var attributesData = typeSymbol.GetAttributes();
+
+			foreach (var attribute in attributesData)
+			{
+				if (IsBoundAttribute(attribute) == BoundFlag.DbBound)
 					return true;
+				foreach (var argument in attribute.NamedArguments)
```

Comment:
See the code style - `if`, `foreach`, etc. must be separated with an empty line.